katalog/x509-exporter/README.md

```diff
@@ -8,14 +8,14 @@ The original project is: [x509-certificate-exporter](https://github.com/enix/x50
 
 ## Requirements
 
-- Kubernetes >= `1.20.0`
-- Kustomize = `v3.3.x`
+- Kubernetes >= `1.21.0`
+- Kustomize = `v3.5.3`
 - [prometheus-operator](../prometheus-operator)
 
 
 ## Image repository and tag
 
-* Certificate exporter image: `registry.sighup.io/fury/enix/x509-certificate-exporter:2.12.1`
+* Certificate exporter image: `registry.sighup.io/fury/enix/x509-certificate-exporter:3.2.0`
```

Comment:
Maybe we can include the update to [3.3.0](https://github.com/enix/x509-certificate-exporter/releases) in monitoring v2? Is just 2 releases ahead with some CVE fixed and does not seem to have breaking changes

Reply:
I took the chance and updated to v3.4.0 while also fixing some mount points to allow the exporter to correctly follow symlinks.